build/Jenkinsfile.pr

```diff
@@ -293,9 +293,16 @@ pipeline {
             dir('src/github.com/mattermost/mattermost-server/build') {
                 ansiColor('xterm') {
                     sh """
+                        # Capture docker logs
+                        docker-compose logs --tail="all" -t --no-color > docker-compose_logs
+                        docker ps -a --no-trunc > docker_ps
+                        docker stats -a --no-stream > docker_stats
+                        zip -qr docker_logs.zip docker-compose_logs docker_ps docker_stats 
```

Comment:
nit: can we `tar` this instead?

Reply:
Done. https://build.mattermost.com/job/mp/job/mattermost-server-pr-new/job/docker_logs/10/artifact/docker_logs.tar.gz